src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -124,7 +124,15 @@ public function handleError($type, $msg, $file, $line, $context = [])
             return \call_user_func(self::getPhpUnitErrorHandler(), $type, $msg, $file, $line, $context);
         }
 
-        $deprecation = new Deprecation($msg, debug_backtrace(), $file);
+        $backtrace = debug_backtrace();
+        $backtraceTriggerDeprecation = $backtrace[2] ?? null;
```

Comment:
can/should we move this logic in the constructor of `Deprecation`?

Reply:
I moved it to the constructor :)